celery/contrib/abortable.py

```diff
@@ -147,9 +147,9 @@ class AbortableTask(Task):
 
     abstract = True
 
-    def AsyncResult(self, task_id):
+    def AsyncResult(self, task_id, **kwargs):
```

Comment:
I am always sceptical about having **kwargs. I know Python developers love to stick it everywhere but question here is do we really need it here when we know ahead it is used only to pass a single parameter...

Reply:
I see, how do I get the name if we know the task id? I'm writing some internal wrappers around `AsyncResult` to measure timing, so its important to know the name of the task in question

Reply:
So, I still haven't figured out how to solve my use-case. The problem I'm facing is the following:

We want to dump a metric for counting how long tasks take as perceived by the clientside when they wait for results. For this, I overrode `celery.AsyncResult` with an extended class with a timer.

When we call a task using `apply_async()`, everything is fine since the `Task:AsyncResult()` method in `celery/app/task.py` appropriately passes in `task_name` argument. This name is useful for dumping a metric/timer and for debugging later on. However, when we send a task using `send_task()`, this doesn't happen, which is what I'm adding in this PR. This avoids the confusion about when the argument is available is not.

Reply:
@dejlek ^